build.assets/Dockerfile

```diff
@@ -28,12 +28,13 @@ RUN apt-get update -y --fix-missing && \
     apt-get -q -y upgrade && \
     apt-get install -q -y --no-install-recommends \
         apt-utils \
+        build-essential \
```

Comment:
This also includes `libc6-dev | libc-dev` so we can drop that one too.

Reply:
I kept `libc6-dev` because I wasn't sure which one `build-essential` would pick. It seemed better to be explicit.